src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +172,19 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_output_element_type(const Napi::CallbackInfo& info) {
+    std::vector<std::string> allowed_signatures = {"(number)"};
```

Comment:
```suggestion
    std::vector<std::string> allowed_signatures;
```
You don't have to initialize allowed signatures